cypress/support/pages/admin_console/components/TablePage.ts

```diff
@@ -0,0 +1,113 @@
+import CommonElements from "../../CommonElements";
+
+export default class TablePage extends CommonElements {
+  tableRowItem;
+  tableRowItemCheckbox;
+  tableHeaderRowItem;
+  static tableSelector: string = "table[aria-label]";
```

Comment:
Type can be inferred here.
```suggestion
  static tableSelector = "table[aria-label]";
```